config-overrides.js

```diff
@@ -0,0 +1,11 @@
+const { override, babelInclude } = require('customize-cra')
+const path = require('path')
+
+module.exports = override(
+  babelInclude([
+    path.resolve(__dirname, 'src'),
+    path.resolve(__dirname, 'node_modules/@walletconnect'),
+    path.resolve(__dirname, 'node_modules/@web3modal'),
+    path.resolve(__dirname, 'node_modules/@metamask/utils/node_modules/superstruct'),
```

Comment:
Should we include this as a dependency, instead of using the one included in metamask? 
(It could fail if metamask is upgraded and stops using that)

Reply:
If we don't include this, the project doesn't work :(

check at https://github.com/decentraland/marketplace/blob/master/webapp/config-overrides.js